chapters/usage-profile.md

```diff
@@ -0,0 +1,326 @@
+# 31 Usage Profile section
+
+## 31.1 Usage Scope field <a name="31.1"></a>
+
+### 31.1.1 Description
+
+Software packages are sometimes generated with specific Build conditions based on package supplier's license conditions and intended usage in addition to original software source code. In those cases, it is preferred for the package supplier to clearly state in the SPDX document the intended usage of the package as envisioned when setting the Build conditions, and/or the inspection conditions for packages may also have to be clearly stated for each package.
```

Comment:
With apologies, I'm not sure I really understand what is meant here by "setting the Build conditions, and/or the inspection conditions". I'm not sure a reader of the spec would really understand the purpose of this field or the other "Usage Scope" fields below with how this is currently written. Is it possible to provide more clarity or examples?

Reply:
Following examples are considered as a premise for describing this fields as a intention of package supplier, to a consumer downstream in the supply chain can mechanically process and retrieve the upstream provider's intent from the SBOM file included in the package.

- "Binary file A contained in this package was built with compile-time option X to use on product Y" 
- "This Package is initial Alpha release, limited verification are applied, do not use in the final product"
- "file A is library file for product X and Y", "script file B is configuration file for library A on product X" and "script file C is configuration file for library A on product Y".